### PR TITLE
Adds example of using AppEngine Flex with OpenCensus and Stackdriver.

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: java/appengine/src/main/java/io/opencensus/appengine/flex/CleanupServlet.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/* Copyright 2016 Google Inc.
```
Comment (songy23) [inline java/appengine/src/main/java/io/opencensus/appengine/flex/CleanupServlet.java]: nit: s/2016/2018. Same for all other files.

File: java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import io.opencensus.tags.Tagger;
+import io.opencensus.tags.Tags;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * A utility class for using OpenCensus Stats in microservices.
+ */
+public class OpenCensusStatsUtil {
```
Comment (songy23) [inline java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java]: nit: public final

File: java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  @SuppressWarnings("deprecation")
+  private static final View CLIENT_LATENCY_VIEW =
+      View.create(
+          View.Name.create("my.org/cloud-storage-client/latency"),
+          "Latency in milliseconds",
+          CLIENT_LATENCY,
+          Distribution.create(BucketBoundaries.create(Arrays.asList(
+              1.0, 2.0, 4.0, 8.0, 16.0, 32.0, 64.0, 128.0, 256.0, 512.0, 1024.0, 2048.0, 4096.0))),
+          Arrays.asList(CLIENT_METHOD),
+          View.AggregationWindow.Cumulative.create());
```
Comment (songy23) [inline java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java]: Window can be removed.

File: java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java (hunk 2 above)
Comment (dinooliva) [reply java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java]: Done.

File: java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java (hunk 3 above)
Comment (dinooliva) [reply java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusStatsUtil.java]: Done.

File: java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusTraceUtil.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import io.opencensus.trace.samplers.Samplers;
+
+import java.io.IOException;
+
+public final class OpenCensusTraceUtil {
+  public static final String PROJECT_ID = ServiceOptions.getDefaultProjectId();
+
+  // Tracing
+  public static final TraceOptions SAMPLED = TraceOptions.builder().setIsSampled(true).build();
+  public static final Tracer tracer = Tracing.getTracer();
```
Comment (songy23) [inline java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusTraceUtil.java]: nit: this can be private, similar to `tagger`.

Comment (dinooliva) [reply java/appengine/src/main/java/io/opencensus/appengine/flex/OpenCensusTraceUtil.java]: Done. And SAMPLED isn't need any longer either.

**Pull request merged**